api/src/test/resources/testAppDataDir/configuration/jsonkeyvalues/config.json

```diff
@@ -21,5 +21,8 @@
     "report.opdconsult.active" : "true",
     "report.opdconsult.diagnosisQuestion.concept" : "95312123-e0c2-466d-b6b1-cb6e990d0d65",
     "report.opdconsult.diagnosesList.concept" : "b3ea209f-9979-481b-9c64-b6dd883c75bf",
-    "report.opdconsult.referredTo.concept" : "96408258-000b-424e-af1a-403919332938"
+    "report.opdconsult.referredTo.concept" : "96408258-000b-424e-af1a-403919332938",
+    "report.newEpisodesOfDiseases.conceptSet" : "c1ee4e1a-96d5-4cb1-82d1-95e367cfee20",
+    "report.newEpisodesOfDiseases.question.concept" : "95312123-e0c2-466d-b6b1-cb6e990d0d65",
```

Comment:
Just to confirm, this question concept will likely hold 2 answers in our case: "Chief Complaint" and "Diagnoses" (that's not the exact lables I guess but that's the idea right?)

Reply:
>Just to confirm, this question concept will likely hold 2 answers in our case

I'd prefer to have a question(s) concept set, meaning a configurable set containing questions to report on.